config.py

```diff
@@ -84,9 +137,9 @@ def device_config(self) -> tuple:
             ):
                 print("Found GPU", self.gpu_name, ", force to fp32")
                 self.is_half = False
-                use_fp32_config()
             else:
                 print("Found GPU", self.gpu_name)
+                use_fp32_config()
```

Comment:
The function name is still `use_fp32_config` so you might want to change its name to `use_fp16_config` for example.